tools/importer-rest-api-specs/components/dataapigeneratorjson/templater_terraform_resource_definition.go

```diff
@@ -0,0 +1,84 @@
+package dataapigeneratorjson
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/hashicorp/pandora/tools/importer-rest-api-specs/models"
+	"github.com/hashicorp/pandora/tools/sdk/resourcemanager"
+)
+
+type ResourceDefinition struct {
+	DisplayName                  string `json:"DisplayName"`
+	Id                           string `json:"Id"`
+	Label                        string `json:"Label"`
+	Category                     string `json:"Category"`
+	Description                  string `json:"Description"`
+	ExampleUsage                 string `json:"ExampleUsage"`
+	GenerateIDValidationFunction bool   `json:"GenerateIDValidationFunction"`
```

Comment:
we're using `Id` throughout the rest of the codebase (for fields - methods use `ID` to differentiate them) - so can we update this to match:

```suggestion
	GenerateIdValidationFunction bool   `json:"GenerateIdValidationFunction"`
```